src/components/Header/index.tsx

```diff
@@ -17,18 +17,21 @@ import {
 import { isMobile } from '../../utils/screen'
 import { AppContext } from '../../contexts/providers/index'
 import { AppDispatch, ComponentActions } from '../../contexts/providers/reducer'
+import Dropdown from './Dropdown'
 
-const MenuDatas = [
-  {
-    name: i18n.t('navbar.charts'),
-    url: '/charts',
-  },
-]
+const menuDatas = () => {
```

Comment:
Why use a function to return the constant?

Reply:
Because we should support English and Chinese switching, when I switch other language, all `i18n` elements should render again.

Use function instead of const to make render success.